doc/ss_logger.rst

```diff
@@ -0,0 +1,144 @@
+****************
+SmartSim Logging
+****************
+========
+Overview
+========
+SmartSim supports experiment tracking with logging functionality
+using the SmartSim log module. The logger, supported by Python logging, enables
+monitoring the experiment at runtime by allowing users to print messages **to stdout**
+and/or **to file** from within the Python driver script. The SmartSim logger allows users to categorize
+messages by severity level. Users may instruct the SmartSim to print
+specified severity levels and ignore others through the `SMARTSIM_LOG_LEVEL`
+environment variable.
+
+The `SMARTSIM_LOG_LEVEL` environment variable accepts **three** inputs,
+ranked from lowest severity (1) to highest (3). Each severity level has
+associated logging functions users may use within an experiment driver script:
+
+    1. level: `quiet`
+       - function: ``logger.error()`` & ``logger.warning()``
+    2. level: `info`
+       - function: ``logger.info()``
+    3. level: `debug`
+       - function: ``logger.debug()``
+
+The messages printed to stdout or to file by the SmartSim logger depend on the specified value.
+The logger prints the specified severity level and all levels below it. For instance,
```

Comment:
Again reversed text from standard usage. If `level == "INFO"`, and it prints warn/error, that is a high-pass filter and would be printing specified severity level and all levels _above_ it.